src/types/guild.ts

```diff
@@ -380,6 +381,90 @@ export interface GuildEmbed {
   enabled: boolean;
 }
 
+/** https://discord.com/developers/docs/resources/audit-log#audit-log-object */
+export interface AuditLogPayload {
+  /** list of webhooks found in the audit log */
+  webhooks: WebhookPayload[];
+  /** list of users found in the audit log */
+  users: UserPayload[];
+  /** list of audit log entries */
+  audit_log_entries: AuditLogEntryPayload[];
+  /** list of partial integration objects */
+  integrations: Partial<Integration>[];
```

Comment:
```suggestion
  integrations: Pick<Integration, "id" | "name" | "type" | "account">[];
```

Reply:
Are you sure that only these parameters exist? And there is no chance of the others be there too?

Reply:
> Are you sure that only these parameters exist? And there is no chance of the others be there too?

Yes and not sure about the second question, maybe you should ask in API server,